Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -506,25 +520,33 @@ def fetch_attributes_command(client: Client, params: Dict[str, str]):
     Returns: List of indicators.
 
     """
+    global LIMIT
```

Comment:
```suggestion
```
you do this line only if you want to set the global.
also, instead of passing it to other methods - you can just use it in them directly